contributors/devel/sig-architecture/godep.md

```diff
@@ -249,3 +249,59 @@ or the Steering Committee (@kubernetes/steering-committee) to ensure that they
 are compatible with the Kubernetes project license. It is also important to note
 and flag if a license has changed when updating a dependency, so that these can
 also be reviewed.
+
+For reference, whitelisted licenses as per the CNCF Whitelist Policy are
+mentioned below.
+
+### License changes
+
+Some licenses are whitelisted as per the CNCF Whitelist Policy
+(adopted by the CNCF Governing Board on 2018-05-01). This whitelisting process is
+described below.
+
+In case of questions or concerns regarding the list of approved licenses,
+please create an issue or send an email to the [SIG Architecture] and
+[SIG Release] mailing lists.
```

Comment:
Added a note about escalating to SIG Arch or SIG Release in case of discussions around approved licenses.

(Thanks to @dims suggesting this on slack!)